### ISSUE TITLE
Customization needs for the fluids

### ISSUE THREAD

**Issue by arouinfar** [phetsims/buoyancy#58]
For #51 and related to the changes to the fluids made in #54

Intro Screen:
- [x] The Fluid RadioButtonGroup will include all non-mystery/custom liquids - Gasoline, Oil, Water, Seawater, Honey, and Mercury. 
- [x] The `visibleProperty` of Oil and Mercury will be false by default.
- [ ] ~Each radio button should have an instrumented `textProperty` so clients can edit the labels.~ MK: I don't think this is the path now. Maybe we should instrument the global Materials so that their `nameProperty` can be edited. Not sure, let's discuss.

All other screens:
- [x] The Fluid ComboBox will contain all possible fluids (all 6 mystery fluids on all screens)
- [x] Use the `visibleProperty` of the listBoxItem to control which mystery items are available on each screen (as specified in #54).

**Comment by zepumph** [comment]
Working on this now.

**Comment by zepumph** [comment]
I made some good progress here, but I think that we are not instrumenting the combo box fully right now, as autoselect isn't working very well. I'll come back to this.

**Comment by zepumph** [comment]
Ok that was actually quite simple. I believe all is complete here and we are ready for review. Instead of assigning this specific issue back to @arouinfar, I'll keep track of phet-io review issues over in #51, and we can do it all in one batch when the time comes.

**Comment by zepumph** [comment]
- [x] Show oil fluid on the compare screen. Thus, all screens will have the 6 non custom/mystery fluids available.

**Comment by AgustinVallejo** [comment]
Pushed a small change. As of now VISIBLE_FLUIDS is the same as Material.BUOYANCY_FLUID_MATERIALS. I wasn't sure if we should delete VISIBLE_FLUIDS and its logic, or keep it in case future customization is wanted. @zepumph Tossing back to hear your thoughts.

**Comment by AgustinVallejo** [comment]
Talked with @zepumph and @samreid and we decided to delete that logic. Self assigning

**Comment by AgustinVallejo** [comment]
Done, back to Amy.

**Comment by arouinfar** [comment]
Thanks @AgustinVallejo, the available fluids and their default visibility looks good on main. Closing.

> The Fluid RadioButtonGroup will include all non-mystery/custom liquids - Gasoline, Oil, Water, Seawater, Honey, and Mercury.
>  The `visibleProperty` of Oil and Mercury will be false by default.

We no longer use a RadioButtonGroup on the Intro Screen/Buoyancy Basics. I confirmed all non-mystery/custom fluids are included in the ListBox. None of the options are hidden by default because we no longer have the space constraints of a RadioButtonGroup.

> MK: I don't think this is the path now. Maybe we should instrument the global Materials so that their nameProperty can be edited. Not sure, let's discuss.

Correct! Thanks for disregarding the request to instrument the `textProperty`. In https://github.com/phetsims/density-buoyancy-common/issues/250, we will add a linked StringProperty under each mystery material in `global`.
